XMGDialogue/DialogueController.cs

```diff
@@ -118,17 +122,28 @@ public class DialogueController {
 
 		#region Dialogue Actions
 
+		/// <summary>
+		/// Registers multiple actions to this dialogue controller.
+		/// </summary>
+		/// <param name="actions">A dictionary of actionKey & actionParam strings.</param>
+		private void RegisterDialogueActions(Dictionary<string, string> actions) {
+			foreach (KeyValuePair<string, string> action in actions) {
+				if (this.eventActionList.ContainsKey(action.Key)) {
+					continue;
+				}
+
+				DialogueActionDelegate newDelegate = DialogueManager.Instance.ParseActionKey(action.Key);
+				this.RegisterDialogueAction(actionTag: action.Key, actionDelegate: newDelegate);
```

Comment:
Unless you are using default parameters and skipping some you should not explicitly call out the parameter name.